client/domain/PromptObjectFactory.js

```diff
@@ -59,6 +59,10 @@ tie.factory('PromptObjectFactory', [
       return this._mainFunction;
     };
 
+    Prompt.prototype.getOutputFunctionName = function() {
+      return this._outputFunction;
```

Comment:
I'm not bothered by it either way, but for consistency we'd probably need to make them all (Input|Transformation|Evaluation|Output)FunctionName in the schema.